doc/api/stream.md

```diff
@@ -695,8 +695,9 @@ Not all [`Readable`][] streams will emit the `'close'` event.
 added: v0.9.4
 -->
 
-* `chunk` {Buffer|string|any} The chunk of data. For streams that are not
-  operating in object mode, the chunk will be either a string or `Buffer`.
+* `chunk` {string|Buffer|Uint8Array|any} The chunk of data. For streams that are not
```

Comment:
Forgive me if this is a naive question, but if it's `any`, then shouldn't it just be `{any}`? As far as I can tell, `stream.md` and one entry in `process.md` are the only places where `{any}` appears in conjunction with other types.

@nodejs/documentation @nodejs/streams

Reply:
Does it differ according to the stream mode (object/non-object)? Should we clarify this if so?

Reply:
Oh, it is clarified below.

Reply:
@Trott I see your point. But it may be easier for people to understand what are the options of  `chunk` type since it will be `any`  only for streams in object mode and `string|Buffer|Uint8Array` for streams with object mode turned off. I checked the places in the docs where `any` is the only type and I haven't found similar situation. Looking forward to reply from more experienced guys

Reply:
In what cases would `Uint8Array` be returned rather than a buffer?

Reply:
> @Trott I see your point. But it may be easier for people to understand what are the options of  `chunk` type since it will be `any`  only for streams in object mode and `string|Buffer|Uint8Array` for streams with object mode turned off. I checked the places in the docs where `any` is the only type and I haven't found similar situation. Looking forward to reply from more experienced guys

Works for me, especially since it's already that way elsewhere in the doc so if someone wants to change it, this will just get handled during that cleanup anyway.

Thanks for the explanation of the reasoning here.

(Small note: There are definitely people on this project who don't consider themselves "guys".)

Reply:
> In what cases would Uint8Array be returned rather than a buffer?

@benjamingr There are no such cases, my bad :(
`Uint8Array` is transformed into `Buffer` at `push` stage, so it can be returned neither by `read()` nor in `'data'` event. I've updated the PR. Thank you for pointing it out!